src/skvhttpclient/CMakeLists.txt

```diff
@@ -1,6 +1,16 @@
 cmake_minimum_required (VERSION 3.13)
 
-project (skvlib VERSION 0.0.1 LANGUAGES CXX)
+project (skvhttpClient VERSION 0.0.1 LANGUAGES CXX)
+
+##
+## MAIN_PROJECT CHECK
+## determine if we are built as a subproject (using add_subdirectory) or as the main project
+##
+set(MAIN_PROJECT OFF)
```

Comment:
Does this still apply? You mentioned you couldn't get the subproject approach to work.

Reply:
It doesn't apply, no. Left over from me trying to get it to work as a subproject. I'll leave it in for now though as it doesn't hurt either and is needed for proper subproject eventually